### DISCUSSION TITLE
Informal guidelines ?

### DISCUSSION THREAD

**Discussion by JBBianchi** [serverlessworkflow/specification#887] [General]
It's wonderful to see the recent surge of activity in our community. The lively exchange of ideas and discussions is a testament to the vitality of our project. However, with this excitement, our conversations can sometimes become a bit scattered. To help us maintain productive and focused discussions, here are a few informal guidelines:

- **Stay on Topic**: When opening an issue or discussion, focus on one specific topic. Try to keep replies relevant to the subject, even if related topics come up.
- **Structured Replies**: Aim to synthesize your ideas into a single, well-structured reply, using quotes as necessary. This reduces the number of fractional responses.
- **Pace Your Responses**: Limit the number of replies per hour/day. Allow some time before responding to give yourself time to formulate your thoughts and to let others contribute.
- **Community Involvement*: For opinionated decisions, consider using polls or inviting feedback from the community. The more input, the better.
- 
And a final, crucial point imo:

- **PR Etiquette**: Open a pull request (PR) only after reaching a consensus. The PR review should focus on ensuring the agreed-upon changes were correctly implemented, not on debating the topic. Discussions should occur before the PR is created.
These are just my suggestions, and no one is required to follow them strictly. However, I believe that by slowing down and considering our responses more carefully, we can have more meaningful and productive discussions.

In our fast-paced society and social media-driven world, it's easy to fall into the trap of instantaneous reactions. While enthusiasm is wonderful, taking the time to reflect and thoughtfully engage can lead to more beneficial and constructive outcomes for our community.

Most importantly, please continue to share your ideas and participate actively. Your enthusiasm and contributions are what make this community thrive. Let's keep the energy positive and collaborative! Thank you all for your time and hard work, it's highly appreciated ❤️

**Comment by fjtirado** [comment]
I think we need to assume good intention in comments. So I want to take benefit of the chance provided by @JBBianchi to apologize if I use some expression that might have given the opposite impression. 
From my side, I would try to be less prolific, write less but clearer messages (they will be necessary longer ;)) and try to not react badly if personally called (or better said, when I felt I have been personally called)
On the other side, I feel once we we have an agreement, we should obey it. If we realize we overlook something, we should have a procedure to amend it in a clear way, to avoid misunderstandings and be conscience that rolling it back will necessarily cause more discussion.

**Reply by fjtirado** [reply]
Also, I propose to use the weekly meeting to review opened issues and try to reach agreeement (normally conversations are more agile)

**Reply by ricardozanini** [reply]
We should make an effort as a community to try to have the meetings every week. I believe it's important to meet face to face and clear out any type of misunderstandings and break the ice. 🥂

**Comment by ricardozanini** [comment]
@JBBianchi why not adding this wording to the CONTRIBUTING.md file?

**Reply by JBBianchi** [reply]
I'm not against it, we surely can. Do you want me to open a PR?

**Reply by bvandewe** [reply]
#dibs :)

https://github.com/bvandewe/sw-specification/commit/9ddc02fc9eb26524314496868c0d2b0fe3621d32

**Reply by JBBianchi** [reply]
@bvandewe any updates on the upcoming PR ?
